tbvaccine/__main__.py

```diff
@@ -0,0 +1,26 @@
+import os
```

Comment:
Hmm, I've never seen this usage before, thank you. This doesn't have any negative effects on importing `tbvaccine` normally, right?